static/event-hub.ts

```diff
@@ -26,7 +26,7 @@ import GoldenLayout from 'golden-layout';
 import * as Sentry from '@sentry/browser';
 
 import {Hub} from './hub';
-import {Events} from './events';
+import {EventsList} from './events-list';
```

Comment:
Maybe EventMap would be a better name here? Events have a name->data mapping which makes Map/Record/etc more suitable in my opinion

Reply:
Sure!